### DISCUSSION TITLE
Icons size

### DISCUSSION THREAD

**Discussion by ccharris940406** [kyazdani42/nvim-web-devicons#280] [Q&A]
Is there a way to resize icons? ty
![image](https://github.com/nvim-tree/nvim-web-devicons/assets/47110071/cdd37eff-0ee1-432d-9faa-7aa13bb5afa2)

**Comment by gegoune** [comment]
Yes, your terminal emulator might be able to control it, depending whether such option is exposed to users. Unfortunately this is completely out of nvim-tree's control.

**Reply by alex-courtis** [reply]
You might like to try an alternate font flavour: https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#icons-not-correctly-shown

**Comment by ccharris940406** [comment]
I'm using suckles terminal, it's probably a problem of my terminal. I tried with alacritty and it was ok.
